lib/utils/ICETelescope.cpp

```diff
@@ -129,8 +129,7 @@ freq_id_t ICETelescope::to_freq_id(stream_t stream, uint32_t ind) const {
         case 8: // 16 ICEBoards (256 elements) e.g. Pathfinder/HIRAX-128
             return stream_id.slot_id + stream_id.link_id * 16 + ind * 128;
         case 16: // 8 ICEBoards (128 elements) e.g. Allenby
-            // TODO: Check this mapping
```

Comment:
Since you removed the TODO to check the mapping, I assume this has been done?

Reply:
Yes, I checked the mapping with JF.